turbine/src/broadcast_stage/broadcast_utils.rs

```diff
@@ -201,6 +201,8 @@ pub(super) fn set_block_id_and_send(
 ) -> Result<()> {
     bank.set_block_id(Some(block_id));
     if bank.is_frozen() {
+        // XXX: this is where the block was received over the network?
```

Comment:
This is only called when:

1. This node produced the block (was leader)
2. Replay finished processing this block already